rpc-server.py

```diff
@@ -129,8 +138,8 @@ def run_rpc_server(port, no_token):
     print(app_port, file=sys.stdout, flush=True)
 
     # Redirect stdout and stderror to log files
-    sys.stdout = open('rpc-server.out', 'w')
-    sys.stderr = open('rpc-server.err', 'w')
+    # sys.stdout = open('rpc-server.out', 'w')
+    # sys.stderr = open('rpc-server.err', 'w')
```

Comment:
yes... I wonder whether we should add some cli argument to specify output files or maybe print to stdout/stderr and whoever launches the rpc server has the responsibility to redirect to a file if he wants to.
Given that is not very clear the best way to go we can discuss somewhere else.
Meanwhile, I'll merge this.